src/grammar.coffee

```diff
@@ -166,8 +166,8 @@ grammar =
   ]
 
   String: [
-    o 'STRING',                                 -> new StringLiteral $1
-    o 'STRING_START Body STRING_END',           -> new StringWithInterpolations $2
+    o 'STRING',                                 -> new StringLiteral $1, quote: $1.quote, initialChunk: $1.initialChunk, finalChunk: $1.finalChunk, indent: $1.indent, double: $1.double
```

Comment:
So this is the method for passing extra data through the parser? Clever, I can get behind this.

This particular object is ambiguous though, it should have braces.